packages/gunshi/src/utils.ts

```
@@ -74,13 +78,13 @@ export async function resolveLazyCommand<G extends GunshiParams = DefaultGunshiP
 export function resolveBuiltInKey<
   K extends string = CommandBuiltinArgsKeys | CommandBuiltinResourceKeys
 >(key: K): GenerateNamespacedKey<K> {
-  return `${BUILT_IN_PREFIX}${BUILT_IN_KEY_SEPARATOR}${key}`
+  return `${BUILD_IN_PREFIX_AND_KEY_SEPARATOR}${key}`
 }
 
 export function resolveArgKey<A extends Args = {}, K extends string = KeyOfArgs<RemovedIndex<A>>>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)